src/Akka.Persistence.Sql.Linq2Db/Journal/DAO/ByteArrayJournalSerializer.cs

```diff
@@ -43,8 +43,7 @@ public ByteArrayJournalSerializer(IProviderConfig<JournalTableConfig> journalCon
                 return "";
             }
 
-            return tags.Aggregate((tl, tr) =>
-                tl + separator + tr);
+            return $"{separator}{string.Join(separator, tags)}{separator}";
```

Comment:
Tags column should be formatted as ";{tag1};{tag2};{tag3};" with separator prefix and postfix